### DISCUSSION TITLE
How do I conditionally set the location of the generated files in schema

### DISCUSSION THREAD

**Discussion by marjoforcado** [prisma/prisma#14348] [Q&A]
### Currently in dev
I have an application that uses multiple _schema.prisma_ files from different subdirectories, so generating all of them in the default directory (which is node_modules) overwrites the other schema. To mitigate the issue I have changed the _output_ value so that the generated files are created within their respective directories.

### Going prod
Now that I'm ready to deploy my application in prod each subdirectory will live in their own docker container and have their own node_modules and so I would like to continue using the default location to generate my files.

### TLDR
How do I dynamically set the _output_ value of _generator client_?

**Comment by well-balanced** [comment]
I don't know if I understand it well, but I think you can set the output field

e.g.

prisma schema
```prisma
// your first schema
generator client {
  provider = "prisma-client-js"
  output   = "[your-default-node-modules-path]/@prisma/main"
}

model User {
  id              Int           @id @default(autoincrement())
  name            String
}

// your second schema
generator client {
  provider = "prisma-client-js"
  output   = "[your-default-node-modules-path]/@prisma/sub"
}

model Post {
  id              Int           @id @default(autoincrement())
  title            String
}
```

your Node.js app
```ts
import { User } from '@prisma/main';
import { Post } from '@prisma/sub';
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
